go/vt/vtgate/vindexes/vschema.go

```diff
@@ -166,6 +178,53 @@ func (col *Column) MarshalJSON() ([]byte, error) {
 	})
 }
 
+type globalSchema struct {
+	keyspaceNames map[string]any
+	tables        map[string]*Table
+	vindexes      map[string]Vindex
+}
+
```

Comment:
it is not clear how they will receive input for global schema to keyspaces mapping.
Should we not just have a single global schema.

Reply:
I'm not sure I'm following the question or the suggestion. Could you elaborate or rephrase?